src/l500/l535-device-options.cpp

```diff
@@ -45,7 +45,9 @@ device_options::device_options( std::shared_ptr< librealsense::context > ctx,
              { RS2_OPTION_ALTERNATE_IR,
               { alternate_ir, "Enable/Disable alternate IR" } },
             { RS2_OPTION_AUTO_RX_SENSITIVITY,
-              { rx_sensitivity, "auto gain" } }  // TODO: replace the description
+              { rx_sensitivity, "auto gain" } },  // TODO: replace the description
```

Comment:
Please replace the description, even something like `Enable automatic sensitivity` is better than this

Reply:
done - took from confluence